fix mobile sidebar dropdown

s-babaeizadeh opened the pull request:
This pull request is related to Fix mobile sidebar link/dropdown #6.

s-babaeizadeh committed:
fix mobile sidebar dropdown

Oslonline commented:
Hi !

Yeah that's what we need but you need to make some little changes :

1 - Ensure that users are redirected directly to the dedicated homepage only on mobile devices. For the desktop version, keep the dropdown menus.
2 - Use the Link component from react-router-dom for redirecting users, instead of using a dropdown and a new function. If necessary, create a custom mini component within the same file to apply the same styles as the dropdowns to the Link component (icon & padding).
3 - Double-check that the route names are correct and match the routes defined in the Route.jsx file and the Header.jsx component.
4 - For this and future PRs, ensure that all unnecessary comments are removed from your code.

Leet me know if you need any further explanations about these changes !

s-babaeizadeh committed:
fix sidebar problem for mobile and desktop

s-babaeizadeh commented:
Hi, I'm sorry for the issues in the previous pull request. In this pull request, the comments that were mentioned have been addressed. If there are any problems, I am happy to work on them.

Oslonline committed:
Update SidebarNav.jsx

Change div with onClick action with Link from react-router-dom.
Add styling when on the main pages of each category.
